### PR TITLE
Updated cses-1753(KNP) with Python Code

### PR DESCRIPTION
**Pull request by nithyabz**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-1753.mdx [hunk 1]
```diff
@@ -59,6 +59,36 @@ int main() {
 ```
 
 </CPPSection>
+<PySection>
+
+```Py
+
+# Computes the Pi array of s
+def pi(s: str) -> List[int]:
```
Comment (SansPapyrus683) [inline solutions/cses-1753.mdx:67]: you didn't import `List`

Comment (nithyabz) [reply solutions/cses-1753.mdx:69]: > 

I have added now.Please review

**Review by SansPapyrus683 — APPROVED** [review]
@ryanchou-dev

**Comment by nithyabz** [comment]
@ryanchou-dev please review

**Comment by SansPapyrus683** [comment]
no need to ping again bruh

**Review by ryanchou-dev — APPROVED** [review]
hmm looks alright

**Comment by ryanchou-dev** [comment]
crap i hate github mobile

**Comment by nithyabz** [comment]
> no need to ping again bruh

Sorry for the inconvenience caused, will take care from next time.

**Pull request merged**